actionpack/lib/action_controller/base.rb

```diff
@@ -128,7 +128,7 @@ module ActionController
   #
   # Action Controller sends content to the user by using one of five rendering
   # methods. The most versatile and common is the rendering of a template.
-  # Included in the Action Pack is the Action View, which enables rendering of ERB
+  # Rails integrates Action View,located in rails/actionview, which enables rendering of ERB
```

Comment:
```suggestion
  # Also included with \Rails is Action View, which enables rendering of ERB
```

I don't think we need to mention where its located, that would be useful to someone working on the framework but not developers reading documentation.